Translation/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -1,169 +1,142 @@
-
-### 1. Build LLM Image
-
-```bash
-git clone https://github.com/opea-project/GenAIComps.git
-cd GenAIComps
-docker build -t opea/llm-textgen:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/llms/src/text-generation/Dockerfile .
 ```
-
-### 2. Build MegaService Docker Image
-
-To construct the Mega Service, we utilize the [GenAIComps](https://github.com/opea-project/GenAIComps.git) microservice pipeline within the `translation.py` Python script. Build MegaService Docker image via below command:
-
-```bash
-git clone https://github.com/opea-project/GenAIExamples
-cd GenAIExamples/Translation/
-docker build -t opea/translation:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f Dockerfile .
+git clone https://github.com/opea-project/GenAIExamples.git
+cd GenAIExamples/Translation/docker_compose/intel/cpu/xeon/
 ```
 
-### 3. Build UI Docker Image
-
-Build frontend Docker image via below command:
+Checkout a released version, such as v1.2:
 
-```bash
-cd GenAIExamples/Translation/ui
-docker build -t opea/translation-ui:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f docker/Dockerfile .
 ```
-
-### 4. Build Nginx Docker Image
-
-```bash
-cd GenAIComps
-docker build -t opea/nginx:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/third_parties/nginx/src/Dockerfile .
+git checkout v1.2
 ```
 
-Then run the command `docker images`, you will have the following Docker Images:
+### Generate a HuggingFace Access Token
 
-1. `opea/llm-textgen:latest`
-2. `opea/translation:latest`
-3. `opea/translation-ui:latest`
-4. `opea/nginx:latest`
+Some HuggingFace resources, such as some models, are only accessible if you have an access token. If you do not already have a HuggingFace access token, you can create one by first creating an account by following the steps provided at [HuggingFace](https://huggingface.co/) and then generating a [user access token](https://huggingface.co/docs/transformers.js/en/guides/private#step-1-generating-a-user-access-token).
 
-## 🚀 Start Microservices
+### Configure the Deployment Environment
 
-### Required Models
+To set up environment variables for deploying Translation service, source the _set_env.sh_ script in this directory:
 
-By default, the LLM model is set to a default value as listed below:
+```
+source ../set_env.sh
+```
 
-| Service | Model             |
-| ------- | ----------------- |
-| LLM     | haoranxu/ALMA-13B |
+The _set_env.sh_ script will prompt for required and optional environment variables used to configure the Translation service. If a value is not entered, the script will use a default value for the same. It will also generate a _.env_ file defining the desired configuration. Consult the section on [Translation Service configuration](#translation-service-configuration) for information on how service specific configuration parameters affect deployments.
```

Comment:
```suggestion
The set_env.sh script will prompt for required and optional environment variables used to configure the Translation service. If a value is not entered, the script will use a default value for the same. It will also generate a .env file defining the desired configuration. Consult the section on [Translation Service configuration](#translation-service-configuration) for information on how service specific configuration parameters affect deployments.
```